sql/core/src/test/scala/org/apache/spark/sql/collation/CollationTypePrecedenceSuite.scala

```diff
@@ -175,6 +175,115 @@ class CollationTypePrecedenceSuite extends QueryTest with SharedSparkSession {
     )
   }
 
+  test("in subquery expression") {
+    val tableName = "subquery_tbl"
+    withTable(tableName) {
+      sql(
+        s"""
+          |CREATE TABLE $tableName (
+          | c1 STRING COLLATE UTF8_LCASE,
+          | c2 STRING COLLATE UNICODE
+          |) USING $dataSource
+          |""".stripMargin)
+
+      sql(s"INSERT INTO $tableName VALUES ('a', 'A')")
+
+      assertImplicitMismatch(
+        sql(
+          s"""
+             |SELECT * FROM $tableName
+             |WHERE c1 IN (SELECT c2 FROM $tableName)
+             |""".stripMargin))
```

Comment:
please, fix indentation in all places

Reply:
Why do we have to do this? This is the way intellij formats multi line strings by default, and looking at the code base it seems like aligning the `|` with the third quote is more often used anyways

Reply:
> This is the way intellij formats multi line strings by default

Just tune IDE once according to the style guide, and the issue will be fixed forever.